pkg/hubble/relay/queue/priority_queue_test.go

```diff
@@ -18,8 +18,12 @@ package queue
 
 import (
 	"testing"
+	"time"
 
 	observerpb "github.com/cilium/cilium/api/v1/observer"
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+	"google.golang.org/protobuf/types/known/timestamppb"
```

Comment:
Please use `github.com/golang/protobuf/ptypes/timestamp` (it re-exports `timestamppb`)